content/api/recipient-lists.apib

```diff
@@ -11,52 +11,81 @@ transmission data, or a stored recipient list id attribute can be specified.
-|multichannel_addresses | JSON array |Array of Multichannel Address objects for a recipient | At a minimum, address or multichannel_addresses is required |If both address and multichannel_addresses are specified only multichannel_addresses will be used.  See [Multichannel Address attributes](#header-multichannel-address-attributes). |
-|return_path | string | Email address to use for envelope FROM | no | Overrides transmission level return_path.<br><br>For <span class="label label-primary"><strong>SparkPost</strong></span> accounts, the domain part of the return_path address must be a [CNAME-verified sending domain](/api/sending-domains/#sending-domains-verify-post).  The local part of the return_path address will be overwritten by SparkPost servers.<br><br>For <a href="https://www.sparkpost.com/enterprise-email/"><span class="label label-warning"><strong>Enterprise</strong></span></a> accounts, the return_path may be any valid email address and the localpart in the return_path will **not** be overwritten by SparkPost servers. |
-|tags | JSON array |Array of text labels associated with a recipient | no | Tags are available in Webhook events.  Maximum number of tags - 10 per recipient, 100 system wide.  Any tags over the limits are ignored.|
-|metadata | JSON object| Key/value pairs associated with a recipient |no | Metadata is available during events through the Webhooks and is provided to the substitution engine.  A maximum of 1000 bytes of merged metadata (transmission level + recipient level) is available with recipient metadata taking precedence over transmission metadata when there are conflicts.  |
-|substitution_data | JSON object | Key/value pairs associated with a recipient that are provided to the substitution engine |no | Recipient substitution data takes precedence over transmission substitution data.  Unlike metadata, substitution data is not included in Webhook events.|
++ Data Structure: Attributes
+    + address (enum, required) - Address information for a recipient. See the [Address Object](#header-address-object).
+        + (object)
+        + (string)
+    + return_path (string) 
+        Email address to use for envelope FROM. Overrides transmission level return_path. <br /> <br /> For <span class="label label-primary"><strong>SparkPost</strong></span> accounts, the domain part of the return_path address must be a [CNAME-verified sending domain](/api/sending-domains/#sending-domains-verify-post).  The local part of the return_path address will be overwritten by SparkPost servers.<br><br>For <a href="https://www.sparkpost.com/enterprise-email/"><span class="label label-warning"><strong>Enterprise</strong></span></a> accounts, the return_path may be any valid email address and the localpart in the return_path will **not** be overwritten by SparkPost servers.
```

Comment:
Let's get rid of the "For SparkPost " and leave that as the default behavior/description. The Enterprise description then shines as different / won't be missed